docs/how-to-install.md

```diff
@@ -6,145 +6,170 @@ license: "This software is licensed under the Apache License version 2."
+Or, you can run the following command to verify that the Gravitino server is running:
 
 ```shell
 curl -v -X GET -H "Accept: application/vnd.gravitino.v1+json" -H "Content-Type: application/json" http://localhost:8090/api/version
 ```
 
-to make sure Gravitino is running.
-
 :::info
-If you need to debug the Gravitino server, enable the `GRAVITINO_DEBUG_OPTS` environment variable in the `conf/gravitino-env.sh` file. Then create a `Remote JVM Debug` configuration in `IntelliJ IDEA` and debug `gravitino.server.main`.
+If you need to debug the Gravitino server, configure the `GRAVITINO_DEBUG_OPTS` environment variable in the `conf/gravitino-env.sh` file.
+Then create a `Remote JVM Debug` configuration in `IntelliJ IDEA` and debug `gravitino.server.main`.
 :::
 
-#### Manage Gravitino Iceberg REST server in Gravitino package
+### Start the Gravitino Iceberg REST server
 
-You can run the Iceberg REST server as either a standalone server or as an auxiliary service embedded in the Gravitino server. To start it as a standalone server, use the command `./bin/gravitino-iceberg-rest-server.sh start` with configurations specified in `./conf/gravitino-iceberg-rest-server.conf`. Alternatively, use `./bin/gravitino.sh start` to launch a Gravitino server that integrates both the Iceberg REST service and the Gravitino service, with all configurations centralized in `conf/gravitino.conf`. 
+You can run the Iceberg REST server either as a standalone server or as an auxiliary service embedded in the Gravitino server.
 
-For more detailed information about the Gravitino Iceberg REST server, please refer to [Iceberg REST server document](./iceberg-rest-service.md).
+- To start the Iceberg REST server as a standalone server, run the `./bin/gravitino-iceberg-rest-server.sh start` command with configurations specified in the `./conf/gravitino-iceberg-rest-server.conf` file.
+- To start the Iceberg REST server as an auxiliary service embedded in the Gravitino server, run the `./bin/gravitino.sh start` command with all configurations in the `conf/gravitino.conf` file.
 
-## Install Apache Gravitino using Docker
+For details about the Gravitino Iceberg REST server, see the [Gravitino Iceberg REST server documentation](./iceberg-rest-service.md).
 
-### Get the Apache Gravitino Docker image
+## Install Apache Gravitino using Docker
 
 Gravitino publishes the Docker image to [Docker Hub](https://hub.docker.com/r/apache/gravitino/tags).
-Run the Gravitino Docker image by running:
+The published Gravitino Docker image only contains the Gravitino server with basic configurations.
 
-```shell
-docker run -d -i -p 8090:8090 apache/gravitino:<version>
-```
+### Prerequisites
 
-Access the Gravitino Web UI by typing `http://localhost:8090` in your browser, or you
-can run
+If you have not installed Docker, download and install it by following the [instructions](https://docs.docker.com/get-started/get-docker/) for your Operating System (OS).
```

Comment:
no need for "(OS)"

Reply:
OS is an abbreviation of operating system. This is common used for technical writing. any special reason why do you want to use operating system?

Reply:
It is unnecessary here as it is not used anywhere else. Usually, you would introduce an abbreviation without capitalisation e.g. "operating system (OS)" and then use OS in the rest of the documents.